metricbeat/module/aerospike/_meta/Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM aerospike:3.9.0
 
 RUN apt-get update && apt-get install -y netcat
-HEALTHCHECK CMD nc -z localhost 3000
+HEALTHCHECK --interval=3s --retries=30 CMD nc -z localhost 3000
```

Comment:
any reason for setting this to 3s instead of one? Every second would probably make the tests quicker.

Reply:
I can unify this to something like `--interval=1s --retries=60`, WDYT?

Reply:
SGTM